### PR TITLE
Loosened DMF for miniAOD taus

### PR DESCRIPTION
**Pull request by andrewj314**

### CODE REVIEW

**Comment by jpavel** [comment]
adding @gpetrucc @arizzi to watch

**Comment by arizzi** [comment]
how many more taus per event show up on ttbar sample?

**Comment by gpetruc** [comment]
Comparing pre and post on 9k ttbar, it seeems there's some increase in taus but the new ones are somewhat smaller (maybe e.g. they have less pf candidates inside), so that in all the increase of size is fairly small (0.45 -> 0.63 kb/event). The slimming of taus had saved us ~0.1 kb/event, so I think it's all acceptable.
http://gpetrucc.web.cern.ch/gpetrucc/micro/tau_pre_pr8421.html
http://gpetrucc.web.cern.ch/gpetrucc/micro/tau_post_pr8421.html

File: PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def miniAOD_customizeCommon(process):
     process.selectedPatJets.cut = cms.string("pt > 10")
     process.selectedPatMuons.cut = cms.string("pt > 5 || isPFMuon || (pt > 3 && (isGlobalMuon || isStandAloneMuon || numberOfMatches > 0 || muonID('RPCMuLoose')))")
     process.selectedPatElectrons.cut = cms.string("")
-    process.selectedPatTaus.cut = cms.string("pt > 18. && tauID('decayModeFinding')> 0.5")
```
Comment (aman0404) [inline PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py]: what is the meaning of decay mode finding in this

Comment (jpavel) [inline PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py]: It means that the hadronic tau has one of the allowed combinations of charged and neutral components - at the time of that release it had been either 1 track and 0/1/2 strips or 3 tracks. The PR by @andrewj314  allowed also other combinations with 2 tracks and strips in order to account for decreased efficiency that came with increased pileup. But anyway, it is almost 2 years old piece of sw, so it is probably outdated, so I suggest contacting tau conveners for more up-to-date info (I left CMS 1.5y ago ;)

**Pull request merged**